packages/plugin-hubspot-analytics/package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "@wasmcloud/plugin-hubspot-analytics",
+  "version": "0.0.0",
```

Comment:
A version might be nice here, even if it's `0.0.1`, though IMO `0.1.0` is also reasonable

Reply:
If we decide to get them out into a registry we can set a version then but for now, this is the convention to indicate that it's a workspace-only package.

Reply:
TIL, thanks!